include/circt/Dialect/SV/SVPasses.td

```diff
@@ -132,4 +132,21 @@ def HWExportModuleHierarchy : Pass<"hw-export-module-hierarchy",
    ];
 }
 
+def HWExportModuleMetadata : Pass<"hw-export-module-metadata",
```

Comment:
It was easier to put this in the SVPasses. If people thing this is a good idea, I'll move it to the HW dialect.